pkg/storage/sqlite/sqlite.go

```diff
@@ -83,6 +83,91 @@ func (s SQLite) ListIndexers(ctx context.Context) ([]*model.Indexers, error) {
 	return indexers, nil
 }
 
+// CreateMovie stores a movie
+func (s SQLite) CreateMovie(ctx context.Context, movie model.Movies) (int32, error) {
+	// log := logger.FromCtx(ctx)
+
+	stmt := table.Movies.INSERT(table.Movies.MutableColumns).RETURNING(table.Movies.ID).MODEL(movie).ON_CONFLICT(table.Movies.ID).DO_NOTHING()
+	result, err := s.handleInsert(ctx, stmt)
+	if err != nil {
+		return 0, err
+	}
+
+	inserted, err := result.LastInsertId()
+	if err != nil {
+		return 0, err
+	}
+	// hope for the best I guess
+	res := int32(inserted)
+	return res, nil
+}
+
+// DeleteMovie removes a movie by id
+func (s SQLite) DeleteMovie(ctx context.Context, id int64) error {
+	stmt := table.Movies.DELETE().WHERE(table.Movies.ID.EQ(sqlite.Int64(id))).RETURNING(table.Movies.ID)
+	_, err := s.handleDelete(ctx, stmt)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// ListMovies lists the stored movies
+func (s SQLite) ListMovies(ctx context.Context) ([]*model.Movies, error) {
+	log := logger.FromCtx(ctx)
+
+	movies := make([]*model.Movies, 0)
+	stmt := table.Movies.SELECT(table.Movies.AllColumns).FROM(table.Movies).ORDER_BY(table.Movies.Added.ASC())
+	err := stmt.QueryContext(ctx, s.db, &movies)
+	if err != nil {
+		log.Errorf("failed to list indexer: %w", err)
+	}
+
+	return movies, nil
```

Comment:
do we want to not handle errors here? I usually let this bubble up as far as it needs to be handled

Reply:
I kept it what ListIndexers was doing. I wasn't sure if the idea was just always return and have it be an empty list if something happened.

Reply:
ah I probably missed that then.. we should probably return an err in all cases